lib/utils.js

```diff
@@ -127,11 +127,51 @@ const colors = {
   green: (message) => `\u001b[1m\u001b[32m${message}\u001b[39m\u001b[22m`
 }
 
+/**
+ * @param {String} packagePath full path to package.json
+ * @param {String} appName name of the app
+ * @description Updates the name property in JSON file
+ * @returns {Promise} 
+ */
+ function updatePackageJson(packagePath, appName) {
+  const packageJSON = require(packagePath);
+  packageJSON.name = appName;
+  return new Promise((resolve, reject) => {
+    fs.writeFile(packagePath, JSON.stringify(packageJSON, null, 2), function(err) {
+      if (err) reject(err);
+      resolve();
+    });
+  });
+}
+
+/**
+ * @param {String} appPath path of the app
+ * @description Creates a list of json files to be modified and invokes the updateFN parllely 
+ * @returns {Promise} 
+ */
+async function setCorrectAppName(appPath) {
+  try {
+    const currentDirectory = process.cwd();
+    const appName = appPath.split(path.sep).pop();
+    const packageFilesRegex = new RegExp(/^(package|package-lock).json$/);
+    const packageFilesList = fs
+    .readdirSync(appPath)
+    .filter(packageFilesRegex.test.bind(packageFilesRegex))
+    .map(file => path.join(currentDirectory, appPath, file));
+
+    const modifyPackagePromiseList = packageFilesList.map(pFile => updatePackageJson(pFile, appName));
+    await Promise.all(modifyPackagePromiseList);
```

Comment:
We can ignore the name field in `package-lock.json` since it will be automatically updated after other npm operations and it doesn't really matter as package.json will be considered as a truth.

So we can have a single `updatePackageJson` call
```suggestion
    await updatePackageJson(path.join(process.cwd(), appPath, 'package.json'), appName)
```

Reply:
Yup if we are changing `package.json` then this can be done

Reply:
Will make these changes and push

Reply:
Made the appropriate changes, have a look